File: examples/hello.java
// FibonacciSum.java
public class hello {
    public static long fibonacciSum(int n) {
        long a = 0, b = 1, total = 0;
        for (int i = 0; i < n; i++) {
            total += a;
            long temp = a;
            a = b;
            b = temp + b;
        }
        return total;
    }

    public static void main(String[] args) {
        System.out.println(fibonacciSum(100));
    }
}

File: examples/hello.js
function fibonacciSum(n) {
    let a = 0, b = 1, total = 0;
    for (let i = 0; i < n; i++) {
        total += a;
        [a, b] = [b, a + b];
    }
    return total;
}

console.log(fibonacciSum(100));
